packages/zoe/src/contractFacet/offerHandlerStorage.js

```diff
@@ -6,6 +6,15 @@ import { canBeDurable, provideDurableWeakMapStore } from '@agoric/vat-data';
 
 import { defineDurableHandle } from '../makeHandle.js';
 
+/**
+ * The following should work. But for some reason, @endo/errors does
+ * not export the type `Details`. TODO fix.
```

Comment:
Would like a link to issue here.

Reply:
Added comment linking to https://github.com/endojs/endo/issues/2339 . Done.